glm/PSPBUILD

```diff
@@ -0,0 +1,30 @@
+# Maintainer: stdgregwar github.com/stdgregwar
```

Comment:
This is optional, so up to you if you want your name here.

Reply:
Okay. So maybe I will remove me